src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -2585,6 +2585,8 @@ void Eltwise::initSupportedPrimitiveDescriptors() {
     };
 
     addDesc(supportedPrimitiveDescriptors, Planar);
+    if (isChannelsFirstApplicable)
+        addDesc(supportedPrimitiveDescriptors, ChannelsFirst);
```

Comment:
Could you please add the `channelsFirst` descriptor (if its applicable) before `planar`? As it is done on L2596-2600.
I'm asking to prioritize `channelsFirst`  layout because in CNN models Convolutional layers are hotspot and use `channelsFirst` layout for more efficient execution. So Eltwise should choose  `channelsFirst`  layout too in some conflict cases to avoid possible Reorders.

Reply:
Sure! I will move `ChannelsFirst` ahead of `Planar` layout!